setup.py

```diff
@@ -28,6 +28,7 @@ def read(filename):
         'cryptography',
         'setuptools_scm',  # for run-time version-detect
         'paramiko',
+        'josepy',
```

Comment:
shouldn't we also increase the minimal acme version?

Reply:
I agree with that. We could put `'acme >= 0.21.0',`.